src/main/java/com/instana/operator/AgentDeployer.java

```diff
@@ -300,14 +303,15 @@ DaemonSet newDaemonSet(InstanaAgent owner,
       container.setResources(new ResourceRequirements());
     }
 
+    final String arch = getArchFromNodeInfo(owner.getMetadata().getNamespace());
```

Comment:
I am not sure about the arch variable. Maybe it would be better to move it to customResourceAdded and set it once.
Or it would be perfect to get the architecture of node where the agent will be run.